bin/sdr-preservation-core_ctl.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+if [ -d ~/sdr-preservation-core/current ]; then
+    cd ~/sdr-preservation-core/current
+else
+    echo "ERROR: There is no application installation in ~/sdr-preservation-core/current"
+    exit 1
+fi
+
+
+# ----
+# The ROBOT_ENVIRONMENT values here should match the config/environment/* files in the shared configs at
+# https://github.com/sul-dlss/shared_configs/tree/sdr-preservation-core_{stage} where
+# {stage} is a capistrano stage, like 'dev|stage|prod'.
+
+HOSTNAME=$(hostname -s)
+case "$HOSTNAME" in
+    'sul-sdr-services')
+        export ROBOT_ENVIRONMENT='production'
+        ;;
+
+    'sdr-services-test' | 'sdr-services-test2')
+        export ROBOT_ENVIRONMENT='stage'
+        ;;
+
+    'sul-sdr-services-dev')
+        export ROBOT_ENVIRONMENT='integration'
+        ;;
+
+    *)
+        echo "WARNING: Failed to identify hostname"
+        export ROBOT_ENVIRONMENT='development'
```

Comment:
should this be integration?  similar to line 26 above?  or maybe just an exit with an error, to be safe?